python/ray/util/client/worker.py

```diff
@@ -596,6 +596,10 @@ def make_client_id() -> str:
     return id.hex
 
 
-def decode_exception(data) -> Exception:
-    data = base64.standard_b64decode(data)
+def decode_exception(e: grpc.RpcError) -> Exception:
+    if e.code() != grpc.StatusCode.INVALID_ARGUMENT:
```

Comment:
Is it possible to match the full list from the reference you provided? Then using another GRPC error code from the list would still be correct.

Reply:
For this case, we should probably explicitly raise anything that isn't specifically the used in `return_exception_in_context`,  otherwise we'll attempt to deserialize details that probably aren't a serialized ray exception, which would cause a deserialization error that might obfuscate the real issue.